README.md

```diff
@@ -78,3 +78,18 @@ checkbox-required
 input-submit
 textarea
 ```
+
+## Options
+
+- `className`: A string or array of string class names.
+- `label`: The intended value of the HTML `label` attribute.
+- `type`: The value of the HTML input `type` attribute.
+- `required`: Value applied to `aria-required` HTML attribute.
+- `hint`: Simliar to a `label`. Applicable to text-based fields such as, `textarea`, `input-text` and `input-date`.
```

Comment:
This adds context to the label, which it is a part of, for input text, radio groups and textarea. It is used within the input by `aria-describedby` for screen readers.